homework-async-and-animations/src/assets/styles/animations.css

```diff
@@ -0,0 +1,27 @@
+.show {
```

Comment:
you can use just transition for simple animations, it's not necessary to create @keyframes